include/dlaf/matrix/copy.h

```diff
@@ -41,9 +41,10 @@ void copy(Matrix<const T, Source>& source, Matrix<T, Destination>& dest) {
 
   for (SizeType j = 0; j < local_tile_cols; ++j) {
     for (SizeType i = 0; i < local_tile_rows; ++i) {
-      pika::dataflow(dlaf::getCopyExecutor<Source, Destination>(),
-                     unwrapExtendTiles(dlaf::matrix::internal::copy_o),
-                     source.read(LocalTileIndex(i, j)), dest(LocalTileIndex(i, j)));
+      ex::when_all(source.read_sender(LocalTileIndex(i, j)),
+                   dest.readwrite_sender(LocalTileIndex(i, j))) |
+          copy(dlaf::internal::Policy<internal::CopyBackend_v<Source, Destination>>{}) |
+          ex::start_detached();
```

Comment:
_This_ closes https://github.com/eth-cscs/DLA-Future/issues/448 (not https://github.com/eth-cscs/DLA-Future/pull/460/files#r812685238). This seems to not be an issue anymore.